src/version.cmake

```diff
@@ -0,0 +1,6 @@
+EXECUTE_PROCESS(
```

Comment:
Pulled your branch - everything works. The problem was that i escaped quotes in match string but you don't need to do that :D